src/test/regress/expected/schema_based_sharding.out

```diff
@@ -384,13 +384,35 @@ SELECT EXISTS(
  t
 (1 row)
 
--- verify that we don't allow creating tenant tables by using CREATE TABLE AS / SELECT INTO commands
+-- verify that we allow creating tenant tables by using CREATE TABLE AS / SELECT INTO commands
 CREATE TABLE tenant_4.tbl_3 AS SELECT 1 AS a, 'text' as b;
-ERROR:  cannot create a tenant table using CREATE TABLE AS or SELECT INTO statements
-CREATE TABLE IF NOT EXISTS tenant_4.tbl_3 AS SELECT 1 as a, 'text' as b;
-ERROR:  cannot create a tenant table using CREATE TABLE AS or SELECT INTO statements
-SELECT 1 as a, 'text' as b INTO tenant_4.tbl_3;
-ERROR:  cannot create a tenant table using CREATE TABLE AS or SELECT INTO statements
+NOTICE:  Copying data from local table...
+NOTICE:  copying the data has completed
+DETAIL:  The local data in the table is no longer visible, but is still on disk.
+HINT:  To remove the local data, run: SELECT truncate_local_data_after_distributing_table($$tenant_4.tbl_3$$)
+CREATE TEMP TABLE IF NOT EXISTS tenant_4.tbl_4 AS SELECT 1 as a, 'text' as b;
+ERROR:  cannot create temporary relation in non-temporary schema
+CREATE UNLOGGED TABLE IF NOT EXISTS tenant_4.tbl_4 AS SELECT 1 as a, 'text' as b WITH NO DATA;
```

Comment:
let's not use WITH NO DATA in one of those so that below query reports some tuples

> -- verify we can query the newly created tenant tables
SELECT * FROM tenant_4.tbl_3 t3 JOIN tenant_4.tbl_4 t4 ON t3.a = t4.a;

Reply:
or maybe change the join query?